honeybadger/provider.go

```diff
@@ -22,17 +22,13 @@ func Provider() *schema.Provider {
 				Optional:    true,
 				DefaultFunc: schema.EnvDefaultFunc("HONEYBADGER_API_KEY", nil),
 			},
-			"team_id": &schema.Schema{
-				Type:        schema.TypeInt,
-				Optional:    true,
-				DefaultFunc: schema.EnvDefaultFunc("HONEYBADGER_TEAM_ID", nil),
-			},
 		},
 		DataSourcesMap: map[string]*schema.Resource{
-			"honeybadger_users": dataSourceUsers(),
+			"honeybadger_teams": dataSourceTeams(),
```

Comment:
If you perform this change, aren't we loosing the datasource related to users?

Reply:
No, `honeybadger_teams`gets also  the user information. Every teams have more than one member